src/pineko/evolve.py

```diff
@@ -419,22 +419,22 @@ def prepare(operator):
                 pid_basis=PyPidBasis.Evol,
             )
             yield (info, op.operator)
-
+    ren_grid2 = xir*xir*mur2_grid
```

Comment:
you can even define it before `alphas_values` so it's more clear for eventual upcoming changes